concepticondata/conceptlists/Gerardi-2020-244.tsv

```diff
@@ -0,0 +1,245 @@
+Gerardi-2020-244-185	185	SOUR/ACID	1906	SOUR	AZEDO / AMARGO
+Gerardi-2020-244-186	186	SPEAK	1623	SPEAK	FALAR
+Gerardi-2020-244-187	187	SPIDER	843	SPIDER	ARANHA
+Gerardi-2020-244-188	188	STAR	1430	STAR	ESTRELA
+Gerardi-2020-244-189	189	STINGRAY	1910	STINGRAY	ARRAIA
+Gerardi-2020-244-190	190	STOMACH	819	STOMACH	ESTÔMAGO
+Gerardi-2020-244-191	191	STONE	857	STONE	PEDRA
+Gerardi-2020-244-192	192	STRONG	785	STRONG	FORTE
+Gerardi-2020-244-193	193	SUCK	1421	SUCK	SUGAR / CHUPAR
+Gerardi-2020-244-194	194	SUN	1343	SUN	SOL
+Gerardi-2020-244-195	195	SWEEP	230	SWEEP	VARRER
+Gerardi-2020-244-196	196	SWEET	717	SWEET	DOCE
+Gerardi-2020-244-197	197	SWEET_POTATO	159	SWEET POTATO	BATATA-DOCE
+Gerardi-2020-244-198	198	SWIM	1439	SWIM	NADAR
+Gerardi-2020-244-199	199	TAIL	1220	TAIL	RABO / CAUDA
+Gerardi-2020-244-200	200	TAPIR	312	TAPIR	ANTA
+Gerardi-2020-244-201	201	TEACH	709	TEACH	ENSINAR
+Gerardi-2020-244-202	202	TELL/NARRATE	1711	TELL	CONTAR
+Gerardi-2020-244-203	203	TERMINATE/FINISH	1766	FINISH	ACABAR / TERMINAR
+Gerardi-2020-244-204	204	TERMITE	883	TERMITE	CUPIM
+Gerardi-2020-244-205	205	TESTICLES	797	TESTICLES	TESTÍCULOS
+Gerardi-2020-244-206	206	THICK	1244	THICK	GROSSO
+Gerardi-2020-244-207	207	THING	738	THING	COISA
+Gerardi-2020-244-208	208	THREE	492	THREE	TRÊS
+Gerardi-2020-244-209	209	THORN	124	THORN	ESPINHO
+Gerardi-2020-244-210	210	THUNDER	1150	THUNDER	TROVÃO
+Gerardi-2020-244-211	211	THROAT	1346	THROAT	GARGANTA
+Gerardi-2020-244-212	212	TICK	1527	TICK	CARRAPATO
+Gerardi-2020-244-213	213	TIE	1917	TIE	ATAR / AMARRAR
+Gerardi-2020-244-214	214	TIRED	1757	TIRED	CANSADO
+Gerardi-2020-244-215	215	TOBACCO	974	TOBACCO	FUMO / TABACO
+Gerardi-2020-244-216	216	TOMORROW	1329	TOMORROW	AMANHÃ
+Gerardi-2020-244-217	217	TONGUE	1205	TONGUE	LÍNGUA
+Gerardi-2020-244-218	218	TREE	906	TREE	ÁRVORE
+Gerardi-2020-244-219	219	TURTLE	112	TURTLE	TARTARUGA / JABOTI / JABUTI
+Gerardi-2020-244-220	220	TWO	1498	TWO	DOIS / DUAS
+Gerardi-2020-244-221	221	UGLY	1548	UGLY	FEIO
+Gerardi-2020-244-222	222	UNCLE(MoBr)	2692	MATERNAL UNCLE (MOTHER'S BROTHER)	TIO (MATERNO)
+Gerardi-2020-244-223	223	UNTIE	352	UNTIE	DESATAR / DESAMARRAR / SOLTAR
+Gerardi-2020-244-224	224	URINATE	592	PISS	URINAR
+Gerardi-2020-244-225	225	VEIN	1924	VEIN	VEIA
+Gerardi-2020-244-226	226	VOMIT	1278	VOMIT	VOMITAR
+Gerardi-2020-244-227	227	VULTURE	1185	VULTURE	URUBÚ
+Gerardi-2020-244-228	228	WALK	1443	WALK	ANDAR / CAMINHAR
+Gerardi-2020-244-229	229	WANT	1784	WANT	QUERER
+Gerardi-2020-244-230	230	WASH	1453	WASH	LAVAR
+Gerardi-2020-244-231	231	WASP	1517	WASP	VESPA
+Gerardi-2020-244-232	232	WATER	948	WATER	ÁGUA
+Gerardi-2020-244-233	233	WE(excl)	1130	WE (EXCLUSIVE)	NÓS (EXCLUSIVO)
+Gerardi-2020-244-234	234	WE(incl)	1131	WE (INCLUSIVE)	NÓS (INCLUSIVO)
+Gerardi-2020-244-235	235	WET	1726	WET	MOLHADO
+Gerardi-2020-244-236	236	WHITE	1235	WHO	QUEM
+Gerardi-2020-244-237	237	WIFE	1199	WIFE	ESPOSA
+Gerardi-2020-244-238	238	WIND	960	WIND	VENTO
+Gerardi-2020-244-239	239	WING	1257	WING	ASA
+Gerardi-2020-244-240	240	WING(2)	1257	WING	ASA
```

Comment:
Is there no difference in meaning between 239 and 240?

Reply:
No difference, also because no entry is mapped to WING(2) in the data. I will remove the mapping, only keeping the English and Portuguese gloss.